Sources/Kitura/CodableRouter.swift

```diff
@@ -539,6 +566,62 @@ public struct CodableHelpers {
         }
     }
 
+    /**
+     * Create a closure that can be called by a codable route handler that
+     * provides an array of tuples of (Identifier, Codable) and an optional `RequestError`
+     *
+     * - Note: This function is intended for use by the codable router or extensions
+     *         thereof. It will create a closure that can be passed to the registered
+     *         route handler.
+     *
+     * - Parameter successStatus: The `HTTPStatusCode` to use for a successful response (see below)
+     * - Parameter response: The `RouterResponse` to which the codable response body (or codable
+     *                       error) and status code will be written
+     * - Parameter completion: The completion to be called after the returned
+     *                         closure completes execution.
+     * - Returns: The closure to pass to the codable route handler. The closure takes two arguments
+     *            `([(Id, OutputType)]?, RequestError?)`.
+     *            If the second (error) argument is `nil` then the first argument (body) should be non-`nil`
+     *            and the response will be considered successful. If the second (error) argument is non-`nil`
+     *            then the first argument is ignored and the response is considered failed.
+     *
+     *            If successful, the HTTP status code will be set to `successStatus` and the `IdentifierCodableArrayResultClosure` output
+     *            will be JSON encoded as an array of dictionaries, which is then sent as the body of the response.
+     *
+     *            If failed, the HTTP status code used for the response will be set to either the
+     *            `httpCode` of the `RequestError`, if that is a valid HTTP status code, or
+     *            `HTTPStatusCode.unknown` otherwise. If the `RequestError` has a codable `body` then
+     *            it will be encoded and sent as the body of the response.
+     */
+    public static func constructTupleArrayOutResultHandler<Id: Identifier, OutputType: Codable>(successStatus: HTTPStatusCode = .OK, response: RouterResponse, completion: @escaping () -> Void) -> IdentifierCodableArrayResultClosure<Id, OutputType> {
+        return {codableOutput, error in
```

Comment:
Don't want to be picky, but there should be a space between `{` and `codableOutput`